diff -u -r a/MODULE.bazel b/MODULE.bazel
--- a/MODULE.bazel	2026-08-26 09:16:10.854960335 +0000
+++ b/MODULE.bazel	2026-08-26 09:45:31.310093878 +0000
@@ -16,12 +16,6 @@
 
 go_sdk = use_extension("//go:extensions.bzl", "go_sdk")
 
-# Don't depend on this repo by name, use toolchains instead.
-# See https://github.com/bazel-contrib/rules_go/blob/master/go/toolchains.rst
-go_sdk.from_file(
-    name = "go_default_sdk",
-    go_mod = "//:go.mod",
-)
 use_repo(
     go_sdk,
     "go_host_compatible_sdk_label",
diff -u -r a/go/private/sdk.bzl b/go/private/sdk.bzl
--- a/go/private/sdk.bzl	2026-08-27 13:39:48
+++ b/go/private/sdk.bzl	2026-08-27 13:39:48
@@ -466,6 +466,8 @@
     goos = ctx.os.name
     if goos == "mac os x":
         goos = "darwin"
+    elif goos == "dragonflybsd":
+        goos = "dragonfly"
     elif goos.startswith("windows"):
         goos = "windows"
 
